recipes/dust/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: dust
+  version: {{ version }}
+
+source:
+  url: https://github.com/bootandy/dust/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 70efd66e662fcd93bbc6cf2f8c3104a1de7e52090f709e9040a34bdc7c72ea9c
+
+build:
+  number: 0
+  script:
+    - RUSTFLAGS="-Clink-args=-Wl,-L$PREFIX/lib" cargo install --path . --root "$PREFIX"  # [unix]
+    - cargo install --path . --root "%LIBRARY_PREFIX%"                                   # [win]
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}  # [unix]
+    - m2w64-binutils       # [win]
+    - cargo-bundle-licenses
```

Comment:
This needs a ru  dependency for the osx deployment target. See the corresponding news item.

Reply:
Understood. I've set both to macOS 10.12.